site/en/docs/extensions/reference/scripting/index.md

```diff
@@ -215,18 +230,48 @@ async function addIframe() {
   return iframe.contentWindow.document.title;
 }
 
-const tabId = getTabId();
-chrome.scripting.executeScript(
-    {
-      target: {tabId: tabId, allFrames: true},
-      func: addIframe,
-    },
-    (injectionResults) => {
-      for (const frameResult of injectionResults)
-        console.log('Iframe Title: ' + frameResult.result);
-    });
+chrome.scripting
+  .executeScript({
+    target: {tabId: getTabId(), allFrames: true},
+    func: addIframe,
+  })
+  .then(injectionResults => {
+    for (const frameResult of injectionResults) {
+      console.log('Iframe Title: ' + frameResult.result);
```

Comment:
Since the code from documentation is often copied by beginners, their actual `result` may be an object or an array of objects that will be rendered as a useless `[object Object]`, so it'd be nice if the documentation would use `,` instead of `+` in all examples.

Reply:
Solid point. I'll update update this PR now to integrate that feedback.